banpools.md

```diff
@@ -12,7 +12,7 @@ Banpools allow for Beemo Max servers to group together and ban users across mult
 
 ---
 
-## How do they work?
+## How does this work?
```

Comment:
This was actually correct before the change, as "they" refers to "Banpools"; "This" no longer fits grammatically.
What we could do, for clarity, is the following change:
```suggestion
## How do Banpools work?
```

Reply:
I was thinking "do banpools "while I wrote something completely different 😂